### PR TITLE
Adding configure_ovn_bgp_agent playbook

### PR DESCRIPTION
**Pull request by jpodivin**

### CODE REVIEW

File: playbooks/configure_ovn_bgp_agent.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Deploy EDPM OVN BGP Agent Configure
+  hosts: all
+  strategy: linear
+  become: true
+  tasks:
+    - name: Configure edpm_ovn_bgp_agent
+      import_role:
+        name: op.edpm.edpm_ovn_bgp_agent
```
Comment (rebtoor) [inline playbooks/configure_ovn_bgp_agent.yaml:9]: `osp.edpm.edpm_ovn_bgp_agent`

File: playbooks/configure_ovn_bgp_agent.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Deploy EDPM OVN BGP Agent Configure
+  hosts: all
+  strategy: linear
+  become: true
+  tasks:
+    - name: Configure edpm_ovn_bgp_agent
+      import_role:
```
Comment (rebtoor) [inline playbooks/configure_ovn_bgp_agent.yaml:8]: `ansible.builtin.import_role`

**Review by rebtoor — CHANGES_REQUESTED** [review]

**Review by rebtoor — APPROVED** [review]
as discussed in chat, let's import the playbook as is then, after verifying that everything is working, we will improve it.

**Review by slagle — APPROVED** [review]

**Comment by jpodivin** [comment]
recheck

**Pull request merged**